src/renderers/shared/utils/ReactErrorUtils.js

```diff
@@ -22,14 +22,14 @@ var caughtError = null;
  * @param {*} a First argument
  * @param {*} b Second argument
  */
-function invokeGuardedCallback<A, B, C>(
+function invokeGuardedCallback<A, B>(
   name: string,
-  func: (a: A, b: B) => C,
+  func: (a: A, b: B) => void,
   a: A,
   b: B,
```

Comment:
This is a great side effect of adding flow types to the codebase, it surfaces how things are really used :p